readme.txt

```diff
@@ -99,6 +99,12 @@ Plugin requires **minimum setup**: after enabling the fee/discount for selected
 
 == Changelog ==
 
+= 2.13.0 - 09/06/2024 =
+* Fix - Added a nonce check for Cross-Site Request Forgery (CSRF) vulnerability on the reset button
+* Fix – Fixed Payment Gateway fees do not change when the payment gateway is changed when WooCommerce Cart/Checkout blocks are set.
+* Fix - Fixed Maximum fee value option is not working as expected.
+* Fix - Fixed A fatal error is shown when activating the 'WooCommerce Table Rate Shipping' plugin with our PGBF lite plugin.
```

Comment:
1. No need to add 'Fixed' in the description. 
2. Change 'is' to 'was'
3. Change point 2 to - Payment Gateway fees did not update when the payment gateway was changed on WooCommerce Cart/Checkout blocks